eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -142,9 +142,20 @@ public EventData(ByteBuffer body) {
     }
 
     /**
-     * Adds an application property associated with this event. If the {@code key} exists in the map, its existing value
+     * Adds an event property associated with this event. If the {@code key} exists in the map, its existing value
```

Comment:
nit:  This may not be clear.   Maybe consider something like "Adds a piece of metadata to the event, allowing publishers to offer additional information to event consumers" ....  or similar?

Reply:
*ctrl+c, ctrl+v*. Thanks. :D